File: Codes_C/Benchmarks/fibonacci.c

#include <stdio.h>
#include <time.h>
#define NTIMES 500000
#define NUMBER 20

int fibonacci(int x);

void main()
{
	long n;
	for (n=1; n<=NTIMES; n++) fibonacci(NUMBER);
}

int fibonacci(int n)
{
	if (n>2) return(fibonacci(n-1)+fibonacci(n-2));
	else return(1);
}

File: Codes_Java/fibonacci.java
public class fibonacci
{

	public static int fibo(int n)
	{
		if (n <= 1) {
			return n;
		}
		else
			return fibo(n-1) + fibo(n-2);
	}

	public static void main(String[] args) {
			for (int in = 0; in < 25; in++) {
				System.out.println(fibo(in));
			}
	}
}